sycl/source/detail/graph_impl.cpp

```diff
@@ -732,6 +754,27 @@ void exec_graph_impl::createCommandBuffers(
   }
 }
 
+exec_graph_impl::exec_graph_impl(sycl::context Context,
+                                 const std::shared_ptr<graph_impl> &GraphImpl,
+                                 const property_list &PropList)
+    : MSchedule(), MGraphImpl(GraphImpl), MPiSyncPoints(), MContext(Context),
+      MRequirements(), MExecutionEvents(),
+      MIsUpdatable(PropList.has_property<property::graph::updatable>()) {
+
+  // If the graph has been marked as updatable then check if the backend
+  // actually supports that. Devices supporting aspect::ext_oneapi_graph must
+  // have support for graph update.
+  if (MIsUpdatable) {
+    bool SupportsUpdate = MGraphImpl->getDevice().has(aspect::ext_oneapi_graph);
+    if (!SupportsUpdate) {
+      throw sycl::exception(sycl::make_error_code(errc::feature_not_supported),
```

Comment:
I'm trying to update sycl graph node but get this "device not support" error on PVC. What device/driver should I use?

Reply:
Please reach out by Email if that hasn't been resolved yet!